src/pd/mod.rs

```diff
@@ -66,6 +68,41 @@ impl RegionStat {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct RegionLeader {
```

Comment:
IMO, this struct means the leader of the region, maybe we should use another name here.

Reply:
How about

```
fn get_region_and_leader(&self, key: &[u8]) -> Result<(Region, Option<Peer>)>
```

Reply:
I don't like tuple too

Reply:
How about `RegionInfo`?

Reply:
I think it is ok